utils/test_helpers.go

```diff
@@ -0,0 +1,11 @@
+package utils
```

Comment:
Can we move all of `proxy/test_helpers.go` to this file? Multiple packages use that so it'd probably be better to have it in a utils package.

Reply:
The method is being used in redis and proxy packages. Moving this to proxy package will cause a cyclic dep failure.

Reply:
@anujjamwal No the other way, can we move all of the test helpers into the `utils` package?

Reply:
Doing so will require a lot of refactoring because that too will cause cycle. The code in test_helper is used in proxy and vis a vis because test_helper returns Proxy instances.

Reply:
Ah makes sense